less_1/h1.py

```diff
@@ -0,0 +1,26 @@
+# Написать функцию (несколько функций), реализующую вывод таблицы
+# умножения размерностью AｘB. Первый и второй множитель должны
+# задаваться в виде аргументов функции.
+from icecream import ic
+
+
+def foo(a, b):
+    """
+1 x 1 = 1
+1 x 2 = 2
+1 x 3 = 3
+2 x 1 = 2
+2 x 2 = 4
+2 x 3 = 6
+3 x 1 = 3
+3 x 2 = 6
+3 x 3 = 9
+"""
```

Comment:
Старайтесь соблюдать отступы даже в докстрингах и комментариях к коду. Это упрощает визуальное восприятие.
```suggestion -8
    """
    1 x 1 = 1
    1 x 2 = 2
    1 x 3 = 3
    2 x 1 = 2
    2 x 2 = 4
    2 x 3 = 6
    3 x 1 = 3
    3 x 2 = 6
    3 x 3 = 9
    """
```